appsec/tests/extension/push_params_block_02.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+Push address gets blocked even when within a hook
```

Comment:
This won't work because blocking within a hook is broken. Do we need this test in this PR?